src/Core/Domain/AbstractBulkCommandHandler.php

```diff
@@ -47,7 +46,7 @@ abstract class AbstractBulkCommandHandler
      *
      * @throws BulkCommandExceptionInterface
      */
-    protected function handleBulkAction(array $ids, string $exceptionToCatch): void
+    protected function handleBulkAction(array $ids, string $exceptionToCatch, mixed $command = null): void
```

Comment:
You should update the PHPDoc to add this `$command` parameter and explain that it can be any type of command used by any handler